### PR TITLE
Switch fieldHelp prop to type Node

### PR DESCRIPTION
**Pull request by danielhartley**
# Description
Updated input-label to change fieldHelp prop from type String to type Node to allow the input to include JSX.

# TODO
- [x] Release notes

### CODE REVIEW

File: src/utils/decorators/input-label/input-label.js [hunk 1]
```diff
@@ -116,9 +116,9 @@ const InputLabel = ComposedComponent => class Component extends ComposedComponen
      * A string representing a help message
      *
      * @property
-     * @type {String}
+     * @type {Node}
      */
-    fieldHelp: PropTypes.string,
+    fieldHelp: PropTypes.node,
```
Comment (patrickmelvin) [inline src/utils/decorators/input-label/input-label.js]: Will this still support strings?

**Comment by toni-leigh** [comment]
You can see here that the demo site information is defined here for all inputs, so the keys in here should be updated alongside the code:

https://github.com/Sage/carbon/blob/6415926e35a11b36fe73feadafadf4cca777a96f/demo/utils/definition/input-definition/input-definition.js

File: src/utils/decorators/input-label/input-label.js (hunk 1 above)
Comment (toni-leigh) [reply src/utils/decorators/input-label/input-label.js]: Node will support anything: https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

File: demo/utils/definition/input-definition/input-definition.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default (definition) => {
   definition.stubAction('onChange', 'value');
 
   definition.propTypes = assign({}, definition.propTypes, {
-    fieldHelp: "String",
+    fieldHelp: "Node",
```
Comment (kylemayne) [inline demo/utils/definition/input-definition/input-definition.js]: Could you edit line 34 from
```
Displays additional text below the input to provide help to the user.
```

to
```
Displays additional information below the input to provide help to the user.
```

**Comment by kylemayne** [comment]
👍

**Pull request merged**